### DISCUSSION TITLE
Version 4.0.98.0-alpha `<Alpha 0>` edition

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/MIR#38] [Announcements]
### Version 4.0.98.0-alpha `<Alpha 0>` edition

* 12 Sep 2022
* For Kerbal Space Program 1.12.3

**Full Changelog**: https://github.com/zer0Kerbal/MIR/compare/4.0.0.0-release...4.0.98.0-alpha

<hr /><em>This discussion was created from the release <a href='https://github.com/zer0Kerbal/MIR/releases/tag/4.0.98.0-alpha'>Version 4.0.98.0-alpha `<Alpha 0>` edition</a>.</em>

**Comment by zer0Kerbal** [comment]
finding the following just floating in the part.cfg @Toa-Sampa , any idea where it should go? SAS? Engine Propellant?

```		RESOURCE
		{
			name = ElectricCharge
			rate = 0.04
		}
```

**Comment by zer0Kerbal** [comment]
Also @Toa-Sampa what bulkheadProfiles are these supposed to be? size1 (1.25m) or ?

**Comment by Toa-Sampa** [comment]
ok, so first, which part in particular are we referring to?
second, I think the parts are generally set to be 2.5 meter diameters in
general, or close to it.

**Reply by zer0Kerbal** [reply]
I've set them all to size1 and some to size1,srf (bulkheadProfiles).

Can't remember which parts off the top of my head, will probably just delete them (the code fragments).

**Comment by Toa-Sampa** [comment]
alright... for the code fragments, I would just disable them inside the CFG
file as well!

**Comment by zer0Kerbal** [comment]
@Toa-Sampa 

Will be closing this thread

new discussion here: https://github.com/zer0Kerbal/MIR/discussions/39 because Alpha-1 prerelease is out with all the parts.

**Comment by Toa-Sampa** [comment]
Got it.
